docs/man/docker-cp.1.md

```diff
@@ -11,8 +12,9 @@ CONTAINER:PATH HOSTPATH
 
 # DESCRIPTION
 Copy files/folders from a container's filesystem to the host
-path. Paths are relative to the root of the filesystem. Files
-can be copied from a running or stopped container.
+path running the command. Paths are relative to the root of
+the filesystem. Files can be copied from a running or stopped
+container.
```

Comment:
First, refer to the parameters using the same format as you provide in the synopsis. This eliminates some ambiguity in the sentence structure.  Let the parameters carry the weight instead of the adjective phrases. Then, don't be afraid of providing a definition.  Don't sacrafice clarity to conciseness.

I am a native speaker. :-D

----

Copy files/folders from a `CONTAINER:PATH` to the `HOSTPATH`. The `HOSTPATH` refers to a location on the host running the command. 
The `CONTAINER:PATH` is relative to the root of the container's filesystem. Files can be copied from a running or stopped container.